libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -57,6 +57,9 @@ export interface IAppHeaderProps {
 
     showUpsellButton?: boolean;
     onUpsellButtonClick?: (e: React.MouseEvent) => void;
+
+    showInviteButton?: boolean;
```

Comment:
Please rename `showInviteButton` → `showInviteItem`, as the item is not a button, is it?